src/components/ScholarshipList.jsx

```diff
@@ -39,6 +40,15 @@ const useStyles = makeStyles((theme) => ({
 
 function ScholarshipList({ scholarships }) {
   const classes = useStyles();
+  const [shareDialogOpen, setShareDialogOpen] = useState(false);
+  const showShareDialog = () => setShareDialogOpen(true);
+  const handleClose = () => setShareDialogOpen(false);
```

Comment:
you should add `share` or `shareDialog` in the name to make it clear this is for the share dialog. E.g. `closeShareDialog`.

Reply:
Ok got it done, changed `handleClose` to `closeShareDialog`